extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationSearch.js

```diff
@@ -0,0 +1,36 @@
+$(function ($) {
+	'use strict';
+
+	var $globalNav = $('.wds-global-navigation'),
+		$searchInput = $globalNav.find('.wds-global-navigation__search-input'),
+		$searchSubmit = $globalNav.find('.wds-global-navigation__search-submit'),
+		placeHolderText = $searchInput.attr('placeholder'),
+		activeSearchClass = 'wds-search-is-active';
+
+	$searchInput.on('focus', function () {
+		if (!$globalNav.hasClass(activeSearchClass)) {
+			$globalNav.addClass(activeSearchClass);
+			$searchInput.attr('placeholder', $searchInput.data('active-text'));
+		}
+	});
+
+	// Make sure default state is correct when navigating back in Chrome
```

Comment:
Can we be more verbose here? @kvas-damian explained to me why it's needed but it wasn't obvious at the first glance.